### DISCUSSION TITLE
How can you discover or list the names of local Talos in Docker clusters?

### DISCUSSION THREAD

**Discussion by jsabo** [siderolabs/talos#9164] [Q&A] [answered]
What is the best way to learn the names of the clusters you've created locally using Talos in Docker? `talosctl cluster show` requires the `--name` argument and defaults to the default cluster.  What's the best way to figure out all the names for your local clusters so you can use talosctl to interact with them?

**Comment by smira** [comment, marked as answer]
`docker ps` should give you an idea, as the container names are prefix with cluster names.

`docker network ls` might also be helpful, as each cluster has its own network.

But please keep in mind that there might be other unrelated containers/networks.
